src/less/ripple/content.less

```diff
@@ -335,9 +335,8 @@ h1 {
 
 // Popover
 .popover {
-  max-width: none;
```

Comment:
Can you confirm the popover still contains the full Ripple address on the trust page?